cdap-security/src/main/java/io/cdap/cdap/security/authorization/AuthorizerClassLoader.java

```diff
@@ -72,7 +78,24 @@ public boolean acceptPackage(String packageName) {
     }
   }
 
-  AuthorizerClassLoader(File unpackedJarDir, @Nullable String authorizerExtraClasspath) {
-    super(unpackedJarDir, authorizerExtraClasspath, createParent(), "lib");
+  AuthorizerClassLoader(File tmpDir, File authorizerExtensionJar,
+                        @Nullable String authorizerExtraClasspath) throws IOException {
+    super(BundleJarUtil.prepareClassLoaderFolder(Locations.toLocation(authorizerExtensionJar), tmpDir),
```

Comment:
You don't need `toLocation`. There is an overloaded method that takes `File`

Reply:
Great. That's better